playground/mocks/data/idp/idx/authenticator-authenticate-remediation-form.json

```diff
@@ -0,0 +1,251 @@
+{
+  "rel": [
+    "create-form"
+  ],
+  "name": "select-authenticator-authenticate",
+  "href": "https://test.okta1.com/idp/idx/challenge",
```

Comment:
Can you replace the domains in your mocks with `localhost:3000`

Example: https://localhost:3000/idp/idx/challenge